planning/behavior_path_planner/src/utils/utils.cpp

```diff
@@ -3147,15 +3147,12 @@ double calcMinimumLaneChangeLength(
   const double & max_lateral_acc = lat_acc.second;
   const double & lateral_jerk = common_param.lane_changing_lateral_jerk;
   const double & finish_judge_buffer = common_param.lane_change_finish_judge_buffer;
-  const auto prepare_length = 0.5 * common_param.max_acc *
-                              common_param.lane_change_prepare_duration *
-                              common_param.lane_change_prepare_duration;
```

Comment:
In https://github.com/autowarefoundation/autoware.universe/pull/4242 , it removes prepare duration if the ego vehicle is near the end of lane.
So, these lines should be removed.

Reply:
It seems there is no degradation!